src/Forem.Api/Internal/ArticleUpdate.cs

```diff
@@ -9,11 +10,34 @@ internal class ArticleUpdate
 	{
 		[JsonProperty("article")]
 		public ArticlePayload Article { get; set; }
+	}
+	public class ArticlePayload
```

Comment:
We can move this class up a folder (into the `Forem.Api` directory). The reason I've got `ArticleUpdate` here is because [this API](https://developers.forem.com/api/v0#tag/articles/operation/updateArticle) is a bit messy to work with so it abstracts that from users of this library.

Reply:
Oh, I see I used to have `ArticlePayload` here too - yeah, still move it up a directory.

Reply:
> We can move this class up a folder (into the Forem.Api directory).

How about we move it to the `Models` folder? The reason for this suggestion is that (most of) the classes in the `Forem.Api` directory are services, unlike the `Models` folder which houses model classes. 

![image](https://user-images.githubusercontent.com/43586181/209906157-748ef520-66d4-421b-8498-9cdca7dc4249.png)

Reply:
Yeah, fair call, let's do that - move the `ArticlePayload` class into the `Models` folder.